client/src/lib/votecounting.ts

```diff
@@ -57,10 +54,10 @@ export function getVotesByPrecinct({
       }
```

Comment:
maybe you're leaving this for a next PR, but the CVR is no longer just a `yes` or ` no`, it's an array, to appropriately express overvotes.

Reply:
Yes, trying to keep this focused.